docs/DeveloperGuide.md

```diff
@@ -387,11 +387,29 @@ otherwise)
 
 **MSS**
 
-1. User wants to look for items that are expiring soon.
+1. User search for items that are expiring soon.
 2. StoreMando returns a list of expiring items.
 
    Use case ends.
 
+**Extensions**
+
+* 1a. User input number is a negative number.
+
+    * 1a1. StoreMando shows an error message.
+    
+    * 1a2. StoreMando prompt the user for a correct input.
```

Comment:
Perhaps, I think this step should be removed.

Reply:
I am just following the format of other use cases.